pkg/arguments/arguments_test.go

```diff
@@ -47,6 +48,13 @@ var _ = Describe("Client", func() {
 				"",
 				"",
 			)
+			flagSet.BoolVarP(
```

Comment:
Should this be hidden or an argv only option? This is just for calling when internal to another command, right?